packages/@n8n/nodes-langchain/nodes/memory/descriptions.ts

```diff
@@ -6,10 +6,10 @@ export const sessionIdOption: INodeProperties = {
 	type: 'options',
 	options: [
 		{
-			// eslint-disable-next-line n8n-nodes-base/node-param-display-name-miscased
-			name: 'Take from previous node automatically',
+			name: 'Connected Chat Trigger Node',
 			value: 'fromInput',
-			description: 'Looks for an input field called sessionId',
+			description:
+				'Looks for an input field called sessionId that is coming from a directly connected Chat Trigger',
```

Comment:
Nit: Does it make sense to wrap the `sessionId` in quotes or backticks to make it standout a bit more?